sceptre/cli/template.py

```diff
@@ -137,3 +137,32 @@ def fetch_remote_template_command(ctx, path):
             output.append(template)
 
     write(output, context.output_format)
+
+
+@click.command(name="stack-name", short_help="Reveal the stack name or names.")
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def stack_name_command(ctx, path):
+    """
+    Show the stack names of all stacks.
+    \f
+
+    :param path: The path to execute the command on.
+    :type path: str
+    """
+    context = SceptreContext(
+        command_path=path,
+        project_path=ctx.obj.get("project_path"),
+        user_variables=ctx.obj.get("user_variables"),
+        options=ctx.obj.get("options"),
+        ignore_dependencies=ctx.obj.get("ignore_dependencies")
+    )
+
+    plan = SceptrePlan(context)
+    responses = plan.stack_name()
+
+    chop_output = len(responses) == 1
+    for response in responses.values():
+        output = response.split(" ")[0] if chop_output else response
+        write(output, context.output_format)
```

Comment:
It seems to me that this command should always output a list, not sequentially output individual values.

Reply:
Honestly, 90% of this PR could go away if you did something like this:

```python
stack_names = []
for stack in plan.command_stacks:
    relative_path = normalize_path(f'{stack.name}.yaml')
    stack_name = stack.external_name
    entry = f'{stack_name} (from {relative_path})'
    stack_names.append(entry)

write(stack_names, context.output_format)
```

Reply:
Agreed, and done.